Content.Shared/Construction/SharedFlatpackSystem.cs

```diff
@@ -147,4 +168,16 @@ public void SetupFlatpack(Entity<FlatpackComponent?> ent, Entity<MachineBoardCom
 
         return cost;
     }
+
+    public Dictionary<string, int> GetFlatpackCreationCostForComputer(Entity<FlatpackCreatorComponent> entity)
+    {
+        var cost = new Dictionary<string, int> {["Glass"] = -200}; // glass for the computer screen
+        foreach (var (mat, amount) in entity.Comp.BaseMaterialCost)
```

Comment:
make a new dictionary on the component called `BaseComputerCost` and rename `BaseMaterialCost` to `BaseMachineCost` or something.

you shouldn't be hardcoding this value in here, even if it's likely to stay static.